django/db/backends/base/base.py

```diff
@@ -74,6 +74,8 @@ def __init__(self, settings_dict, alias=DEFAULT_DB_ALIAS):
         # Tracks if the transaction should be rolled back to the next
         # available savepoint because of an exception in an inner block.
         self.needs_rollback = False
+        # Tracks if we check transaction durability in atomic blocks.
+        self.ensure_durability = True
```

Comment:
Why would we need this on a per-connection basis? Since it's only for the purpose of testing, could it not instead be a single class variable like `Atomic.ensure_durability`?

Reply:
This was just the way I thought of first. Happy to change it.